src/worker.js

```diff
@@ -70,20 +65,19 @@ function threadAlert() {
   var text = Array.prototype.slice.call(arguments).join(' ');
   postMessage({cmd: 'alert', text: text, threadId: selfThreadId});
 }
-out = threadPrint;
-err = threadPrintErr;
+this.err = threadPrintErr;
```

Comment:
i think this should be `var err`?

(the `this.alert` below it is intentionally adding alert to the global namespace, as if this were on the main thread, but `out/err` are just JS locals)

Reply:
Ah, fair enough.